docs/book/src/multiversion-tutorial/api-changes.md

```diff
@@ -12,8 +12,8 @@ schedule: "*/1 * * * *"
 That's a pretty textbook example of a special string format (it's also
 pretty unreadable unless you're a Unix sysadmin).
 
-Let's make it a bit more structured.  According to the our [CronJob
-code][cronjob-sched-code], we support "standard" Cron format.
+Let's make it a bit more structured.  According to our [CronJob
+code][https://github.com/kubernetes-sigs/kubebuilder/blob/master/docs/book/src/multiversion-tutorial/testdata/project/api/v2/cronjob_types.go], we support "standard" Cron format.
```

Comment:
We need to use alias as above [cronjob-sched-code]
Then, define the links at the bottom.

Could you please fix this one?
Also, did you check this change in the preview? 
It probably is not broken

Reply:
Didn't notice this thank you! I reverted it using the alias.